package.json

```diff
@@ -12,8 +12,9 @@
   },
   "dependencies": {
     "@headlessui/react": "^1.7.17",
+    "axios": "^1.6.2",
```

Comment:
Why are we adding this? In general, installing a new dependency requires a justification. Think the more dependencies a project has, the higher number of things are out of our control, so we need to add them sparingly.

Reply:
I needed to add this dependency to use axios

Reply:
removed